sopel/bot.py

```diff
@@ -777,3 +788,13 @@ def reply(self, message, destination=None, reply_to=None, notice=False):
         if reply_to is None:
             reply_to = self._trigger.nick
         self._bot.reply(message, destination, reply_to, notice)
+
+    def kick(self, nick, channel=None, message=None):
+        if channel is None:
+            if self._trigger.is_privmsg:
+                return
```

Comment:
This should raise an error.

Reply:
What error would you like it to raise?

Reply:
A `RuntimeError` is a good start, since this error happens when the developer mess up with the code.